src/python/T0/RunConfig/RunConfigAPI.py

```diff
@@ -616,19 +616,13 @@ def configureRunStream(tier0Config, run, stream, specDirectory, dqmUploadProxy):
             specArguments['TimePerEvent'] = streamConfig.Express.TimePerEvent
             specArguments['SizePerEvent'] = streamConfig.Express.SizePerEvent
 
-            if streamConfig.Express.Scenario == "HeavyIonsRun2":
-                baseMemory = 3000
-                perCoreMemory = 1300
-            else:
-                baseMemory = 3500
-                perCoreMemory = 1050
-
-            specArguments['Memory'] = baseMemory + perCoreMemory
-
+            baseMemory = 500
```

Comment:
From Dima's comment on Slack, I understand that we don't want to assign this 500MB base memory. We want to assign 2GB/core, and use additianal memory only in case of emergency.

Reply:
As mentioned during the meeting we should fit CMS infrastructure requirements that means 2GB/core for any type of job. With the previous formula Tier0 was given in some cases more than 2GB/core so I will start  to test this PR with the runs from 2021 and start to validate which jobs were requiring more than 2GB/core